### PR TITLE
Update pansharpen doc with s2 compatibility

### PR DESCRIPTION
**Pull request by thaisbendixen**
Pull Request
============

> Tracked in JIRA by https://up42.atlassian.net/browse/UP-5410?atlOrigin=eyJpIjoiMmY0MDE0YTI2NGNmNGM1MDg4Y2YyN2U2MWM4MjUwNTMiLCJwIjoiaiJ9

### Scope of the PR

This PR adds information about new compatibility of the Pansharpening Block with S2 Analytic Block.

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — APPROVED** [review]

File: source/up42-blocks/processing/pansharpen.rst [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-Pan-sharpening Pléiades/SPOT
-============================
+Pan-sharpening
+==============
 For more information, please read the `block description <https://marketplace.up42.com/block/903f0435-d638-475e-bbe9-53b5664a22a8>`_.
 
 Block type: ``PROCESSING``
 
-This block pansharpens images of the Pleiades or SPOT sensor. It creates a single high-resolution color image from the high-resolution panchromatic and lower resolution multispectral image bands.
+This block pansharpens images of the Pleiades, SPOT or Sentinel-2 sensors. It creates a single high-resolution color image from high-resolution panchromatic and lower resolution multispectral image bands.
```
Comment (markusuwe) [inline source/up42-blocks/processing/pansharpen.rst:13]: We probably need to explain that in the S2 case this panchromatic band is generated.

File: source/up42-blocks/processing/pansharpen.rst [hunk 2]
```diff
@@ -86,10 +86,48 @@ product appended with the panchromatic band which is clipped to the specific AOI
...
+          "clip_to_aoi":true
+       }
+    }
 
 Advanced
 --------
 
+Synthetic panchromatic band Sentinel-2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Sentinel-2 provides a high range of multispectral bands with different spatial resolutions (10, 20 and 60 m). Since there is no panchromatic (PAN) band in Sentinel-2 images, we use a synthetic panchromatic band to increase the spatial
```
Comment (markusuwe) [inline source/up42-blocks/processing/pansharpen.rst:128]: Ah, here is the explanation. Then a short hint earlier may suffice.

File: source/up42-blocks/processing/pansharpen.rst [hunk 3]
```diff
@@ -169,6 +207,9 @@ Example of parameters to use in the pansharpening block with the ``Esri`` method
     }
 
 
+.. note::
+  It's not recommended to use the Esri pan-sharpening method with **Sentinel-2** data.
```
Comment (markusuwe) [inline source/up42-blocks/processing/pansharpen.rst:211]: Only out of my curiosity: why is that?

**Review by markusuwe — APPROVED** [review]
LGTM

File: source/up42-blocks/processing/pansharpen.rst (hunk 3 above)
Comment (thaisbendixen) [reply source/up42-blocks/processing/pansharpen.rst:211]: For the esri pan-sharpening we need weights for each band. If no weights are given the esri pansharpening method makes all pansharpened bands looks the same. I could not find any exemplary weights for pansharpening s2 data with the esri method, hence can't provide defaults or good results.

**Review by seulgi-up42 — APPROVED** [review]
LGTM! :)

**Pull request merged**